parser_library/src/parsing/grammar/macro_operand_rules.g4

```diff
@@ -53,58 +53,68 @@ mac_preproc[int* paren_count]
 		(
 			ORDSYMBOL (ORDSYMBOL|NUM)*
 			|
-			LPAR {++*$paren_count;}
+			LPAR
 			|
 			AMPERSAND
 		)
 		|
-		LPAR {++*$paren_count;}
+		LPAR
 		|
-		RPAR {--*$paren_count;}
+		RPAR
 		|
 		APOSTROPHE
 		(~(APOSTROPHE|ATTR|CONTINUATION))*
 		(APOSTROPHE|ATTR)
 		|
 		ATTR
 		(
-			{!is_previous_attribute_consuming(*$paren_count == 0, _input->LT(-2))}?
-			(~(APOSTROPHE|ATTR|CONTINUATION))*
-			(APOSTROPHE|ATTR)
-		)?
+			{is_attribute_consuming(_input->LT(-2))}?
+			(
+				ORDSYMBOL
+				|
+				ORDSYMBOL
+				(APOSTROPHE|ATTR)
+				|
+				AMPERSAND
+				ORDSYMBOL
+				(ORDSYMBOL|NUM)*
+				(APOSTROPHE|ATTR)?
+				|
+				EQUALS
+				ORDSYMBOL
+				|
+				(ASTERISK|MINUS|PLUS|LT|GT|SLASH|VERTICAL|IDENTIFIER|NUM|DOT|LPAR|RPAR)
+				(~(APOSTROPHE|ATTR|CONTINUATION))*
+				(APOSTROPHE|ATTR)?
+			)
+			|
+			{!is_attribute_consuming(_input->LT(-2))}?
+			(
+				(~(APOSTROPHE|ATTR|CONTINUATION))*
+				(APOSTROPHE|ATTR)
```

Comment:
maybe
```suggestion
				(~(APOSTROPHE|ATTR|CONTINUATION)|(APOSTROPHE|ATTR)(APOSTROPHE|ATTR))*
				(APOSTROPHE|ATTR)
```
?
anything but apostrophe, or double apostrophe?

Reply:
It helped 👍